lib/reduceBy/reduceBy.test.js

```diff
@@ -1,21 +1,31 @@
+import always from '../always'
+import cond from '../cond'
+import lt from '../lt'
 import reduceBy from '.'
+import T from '../T'
 
-test('groups up reduced values by callback from 3rd argument', () => {
-  var reduceToNamesBy = reduceBy((acc, student) => acc.concat(student.name), [])
-  var namesByGrade = reduceToNamesBy(function(student) {
-    var score = student.score
-    return score < 65
-      ? 'F'
-      : score < 70 ? 'D' : score < 80 ? 'C' : score < 90 ? 'B' : 'A'
-  })
-  var students = [
+const lt_ = than => what => lt(what, than)
+
+it('groups up reduced values by callback from 3rd argument', () => {
+  const reduceToNamesBy = reduceBy((names, { name }) => {
+    names.push(name)
+    return names
+  }, [])
```

Comment:
without this fix there will be https://github.com/nanoutils/nanoutils/issues/176